### ISSUE TITLE
plot2D fcol = NULL and cex and pch

### ISSUE THREAD

**Issue by lmsimp** [lgatto/pRoloc#83]
The arguments `cex` and `pch` do not work when `fcol = NULL`

**Comment by lgatto** [comment]
`pch` does not work

**Comment by lgatto** [comment]
```r
plot2D(hyperLOPIT2015, fcol = "final.assignment", fpch = "markers")
```
doesn't work

**Comment by lgatto** [comment]
See also #30

**Comment by lgatto** [comment]
`plot2D(hyperLOPIT2015, fcol = NULL, pch = 19)` fixed in 6c397accc5c314d9b92f1636d3ed48ccee3470dd
